content/blocksPages/home.json

```diff
@@ -141,6 +141,7 @@
           "text": "See content changes in real-time as you edit",
           "icon2": "FaClock",
           "videoSrc": "/video/key-features/2024-09-13-key-features-visual-editing-v3.webm",
+          "mobileVideoSrc": "/video/key-features-gif-prod/visual-editing.gif",
```

Comment:
Would potentially look at making these .mp4's instead of .gif due to size of requests

Reply:
This could be later work – once the site is working we can ask Landon about making mp4 versions.